R/tax_range_time.R

```diff
@@ -132,6 +144,7 @@ tax_range_time <- function(occdf,
                         taxon_id = seq(1, length(unique_taxa), 1),
                         max_ma = rep(NA, length(unique_taxa)),
                         min_ma = rep(NA, length(unique_taxa)),
+                        group = rep(NA, length(unique_taxa)),
```

Comment:
Should we add some cleanup at the end to remove this column if `group = NULL`? Otherwise we're just getting a column of NAs.